gradle/dependenciesMetadataRules.gradle.kts

```diff
@@ -0,0 +1,230 @@
+import org.gradle.api.artifacts.dsl.ComponentMetadataHandler
```

Comment:
:x: Imports should come after file header.

Reply:
Sorry I did not see that. We do not need to import anythings actually. IDEA is doing weird stuff for gradle.kts files sometimes. It's importing things before it correctly analysed the file... I guess I just have to disable automatic importing? Or what would be your suggestion to avoid this?